doc/developer-guides/hld/hld-devicemodel.rst

```diff
@@ -25,33 +25,33 @@ framework. There are 3 major subsystems in Service VM:
 
 -  I/O Path in Service VM:
 
-   -  HV initializes an I/O request and notifies HSM driver in Service VM
-      through upcall.
+   -  Hypervisor initializes an I/O request and notifies the HSM driver in the
+      Service VM through upcall.
    -  HSM driver dispatches I/O requests to I/O clients and notifies the
       clients (in this case the client is the DM, which is notified
-      through char device)
-   -  DM I/O dispatcher calls corresponding I/O handlers
-   -  I/O dispatcher notifies HSM driver the I/O request is completed
-      through char device
-   -  HSM driver notifies HV on the completion through hypercall
-   -  DM injects VIRQ to User VM frontend device through hypercall
+      through char device).
+   -  DM I/O dispatcher calls corresponding I/O handlers.
+   -  I/O dispatcher notifies the HSM driver that the I/O request is completed
+      through char device.
+   -  HSM driver notifies the hypervisor on the completion through hypercall.
+   -  DM injects VIRQ to the User VM frontend device through hypercall.
 
--  HSM: Hypervisor Service Module is a kernel module in Service VM as a
-   middle layer to support DM. Refer to :ref:`virtio-APIs` for details
+-  HSM: Hypervisor Service Module is a kernel module in the Service VM and is a
+   middle layer to support the DM. Refer to :ref:`virtio-APIs` for details.
 
-This section introduces how the acrn-dm application is configured and
+This section introduces how the ``acrn-dm`` application is configured and
 walks through the DM overall flow. We'll then elaborate on device,
 ISA, and PCI emulation.
 
 Configuration
 *************
 
-The acrn-dm runs using these command line configuration
+The ``acrn-dm`` runs using these command-line configuration
 options:
 
 .. code-block:: none
 
-  acrn-dm [-hAWYv] [-B bootargs] [-E elf_image_path]
+   acrn-dm [-hAWYv] [-B bootargs] [-E elf_image_path]
                [-G GVT_args] [-i ioc_mediator_parameters] [-k kernel_image_path]
                [-l lpc] [-m mem] [-r ramdisk_image_path]
                [-s pci] [-U uuid] [--vsbl vsbl_file_name] [--ovmf ovmf_file_path]
```

Comment:
We have removed `UUID` in ACRN, now may be a good time to also remove it from this doc.

```suggestion
               [-s pci] [--vsbl vsbl_file_name] [--ovmf ovmf_file_path]
```

There is another such instance further below.